654. Maximum Binary Tree.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * Definition for a binary tree node.
+ * function TreeNode(val) {
+ *     this.val = val;
+ *     this.left = this.right = null;
+ * }
+ */
+/**
+ * @param {number[]} nums
+ * @return {TreeNode}
+ */
+let constructMaximumBinaryTree = function(nums) {
+    if (nums.length === 0) {
+        return null
+    }
+    
+    let big = Math.max.apply(Math, nums) //find biggest
+    let index = nums.indexOf(big) // get index of biggest
+    let node = new TreeNode(big) // root is biggest
+    let larr = nums.slice(0, index) // left is everything left of index
```

Comment:
we only use `larr` and `rarr` in one place each, so can we just substitute those there?
`node.left = constructMaximumBinaryTree(nums.slice(0, index))` for ex.